### ISSUE TITLE
[bug]: Repository not found

### ISSUE THREAD

**Issue by Kritiqual** [luong-komorebi/z-a-meta-plugins#10]
### Describe the bug

For testing, I used the snippet in the README.md, and it throw this error:

> ==> Downloading annexes+con (at label: annexes+con)
> Cloning into '/home/kritiqual/.local/zinit/plugins/annexes+con'...
> remote: Not Found
> fatal: repository 'https://github.com/annexes+con/' not found
> 
> Clone failed (code: 128).

### Steps to reproduce

Just like the description

### Expected behavior

Installing all 22 plugins

### Operating System & Version

linux-gnu | pc | x86_64 | x86_64 | x86_64 unknown

### Zsh version

zsh 5.9 (x86_64-pc-linux-gnu)

### Terminal emulator

xterm-kitty

### If using WSL on Windows, which version of WSL

None

**Comment by kbpersonal** [comment]
Yeah same - it's a bit frustrating that the README snippet doesn't give any clue on how to actually use this
